src/main/resources/templates/UpdateUserProfile.html

```diff
@@ -28,8 +28,37 @@
     <li><a href="/Settings">Settings</a></li>
     <li style="float:right"><form th:action="@{/logout}" method="post"><input type="submit" value="Log Out"/></li>
 </ul>
+<div class="row">
+    <div class="q-column">
+        <p>Current Username: [[${#httpServletRequest.remoteUser}]]</p>
+        <p>Current Email Address: </p>
```

Comment:
We have to make sure not to allow users to change their username and email address.